templates/docs/getting-started.md

```diff
@@ -46,7 +46,7 @@ That will generate a whole new Buffalo application that is ready to go. It'll ev
 
 ```text
 $ buffalo new coke
-Buffalo version 0.5.0
+Buffalo version 0.7.1
```

Comment:
Would you mind changing this line to `Buffalo version {{version}}`, that way it will always update. Thanks!